Bhagyashri/Project-WHO/WHO website/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE HTML>
+<html>
+ <head>
+   <title> WHO World Health Oraganization </title>
+   <link rel="shortcut icon" href="/image/favicon.ico" type="image/x-icon">
+   <link rel="stylesheet" type="text/css" href="./css/first.css"/>
+ </head>
+ <body>
+ <section class="sec-first">
+ <header>
+ <hgroup>
+  <nav class="nav-first">
+	  <a href=""> Global </a> &nbsp&nbsp&nbsp&nbsp&nbsp
+	  <a href=""> Region </a>
+  </nav>
+ </hgroup>
+ <hgroup>
+  <nav class="nav-second">
+	  <a href=""> English </a> &nbsp&nbsp&nbsp&nbsp&nbsp
+	  <a href=""> Francies </a>&nbsp&nbsp&nbsp&nbsp&nbsp
+	  <a href=""> Espanol </a>&nbsp&nbsp&nbsp&nbsp&nbsp
+	  <input type="search"/>
+  </nav>
+ </hgroup>
+ </header>
+ </section>
+ <section class="sec-second">
+  <div class="div-first">
+	  <img src="./image/whologo.png" alt="WHO logo" width="280px" height="70px"/>
```

Comment:
don't add width and height to element insted use the css property to apply the width and height.